src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -25,23 +25,40 @@ const DeleteWorkspaceModal = ({ workspace: { workspace: { namespace, name, bucke
   const [workspaceBucketUsageInBytes, setWorkspaceBucketUsageInBytes] = useState()
 
   const signal = useCancellation()
+  const isGoogleWorkspace = !!googleProject
 
   useOnMount(() => {
     const load = Utils.withBusyState(setLoading, async () => {
-      const [currentWorkspaceAppList, { acl }, { usageInBytes }] = await Promise.all([
-        Ajax(signal).Apps.listWithoutProject({ creator: getUser().email, saturnWorkspaceName: name }),
-        Ajax(signal).Workspaces.workspace(namespace, name).getAcl(),
-        Ajax(signal).Workspaces.workspace(namespace, name).bucketUsage()
-      ])
-      setApps(currentWorkspaceAppList)
-      setCollaboratorEmails(_.without([getUser().email], _.keys(acl)))
-      setWorkspaceBucketUsageInBytes(usageInBytes)
+      if (isGoogleWorkspace) {
+        const [currentWorkspaceAppList, { acl }, { usageInBytes }] = await Promise.all([
+          Ajax(signal).Apps.listWithoutProject({ creator: getUser().email, saturnWorkspaceName: name }),
+          Ajax(signal).Workspaces.workspace(namespace, name).getAcl(),
```

Comment:
@marctalbott it's not clear to be that we shouldn't be checking for collaborator emails for Azure workspaces.

Reply:
Ah right, meant to leave a comment on this. The reason we're not checking collaborator emails right now is because this call to Rawls returns a 500 because WSM workspaces don't have all the same expected policies in Sam as a Rawls workspace. https://broadworkbench.atlassian.net/browse/WOR-197 will fix that, but I figured we'd skip the call until it was fixed.

Still, this call to Rawls failing doesn't actually show any errors in the UI, so we could just make the call to Rawls and accept that there will be a silent error until WOR-197 is completed.

Reply:
OK, that's fine. Can you please add a note to WOR-197 though so that we know to come back here and fix this?